internal/install/execution/go_task_recipe_executor.go

```diff
@@ -137,15 +137,15 @@ func (re *GoTaskRecipeExecutor) Execute(ctx context.Context, m types.DiscoveryMa
 	return nil
 }
 
-func varsFromProfile() (types.RecipeVars, error) {
+func varsFromProfile(licenseKey string) (types.RecipeVars, error) {
 	defaultProfile := credentials.DefaultProfile()
 	if defaultProfile.LicenseKey == "" {
```

Comment:
This check probably no longer makes sense if we want to move ahead with this change.

Reply:
Good catch, I've updated the assertion to validate with the provided license key